lib/Math/Matrix.pm6

```diff
@@ -181,6 +188,18 @@ method is-orthogonal(Math::Matrix:D: --> Bool) {
     self.dotProduct( self.T ) ~~ Math::Matrix.identity( $!row-count );
 }
 
+
+method is-positive-definite (Math::Matrix:D: --> Bool) { # with Sylvester's criterion
+    return False unless self.is-square or min self.size < 1;
```

Comment:
why min self.size < 1 ?
Normal we should never have a size = 0 , neither line nor column, i will be at least 1